### PR TITLE
44077 merging tk-config-flame Flame changes into tk-config-default

### PR DESCRIPTION
**Pull request by gomezll**
A few questions.

The info.yml core version was far older in default. Was wondering if it is appropriate to updated it. Or if i should leave it alone. Is this OK?

There are changes referencing flame-premium that were commented out. I was wondering if i should bring them over or not include them. I did the former here. Is this OK?

The path version referenced is 18.3 as in tk-config-flame. tk-config-default2 references 18.2 which is one minor version older. Just checking that using 18.3 is OK?

### CODE REVIEW

File: info.yml [hunk 1]
```diff
@@ -16,4 +16,4 @@ description: "A VFX Example Config and our default configuration."
 
 # Required minimum versions for this item to run
 requires_shotgun_version: "v6.2.0"
-requires_core_version: "v0.18.18"
+requires_core_version: "v0.18.45"
```
Comment (gomezll) [inline info.yml]: Is this core version update OK? Or is it too risky?

File: env/includes/app_launchers.yml [hunk 2]
```diff
@@ -48,6 +48,98 @@ launch_3dsmax:
...
+    name: tk-multi-launchapp
+  mac_args: ''
+  mac_path: '@flame_mac'
+  menu_name: Launch Flame
+  versions: []
+  windows_args: ''
+  windows_path: ''
+#
+# -------------------------------------------------
+# Flame Premium
```
Comment (gomezll) [inline env/includes/app_launchers.yml]: Flame premium changes are commented out. Should i exclude them?

File: info.yml (hunk 1 above)
Comment (josh-t) [reply info.yml]: Check with Francis on this.

Comment (gomezll) [reply info.yml]: Francis said this was OK. But that we will probably want to update to an even newer version once all the app-store has been updated with latest changes.

**Pull request merged**